common/display/virtualpanoramadisplay.cpp

```diff
@@ -492,6 +522,29 @@ int VirtualPanoramaDisplay::GetDisplayPipe() {
   return -1;
 }
 
+#ifdef HYPER_DMABUF_SHARING
+bool VirtualPanoramaDisplay::SetHyperDmaBufMode(uint32_t mode) {
+  if (hyper_dmabuf_mode_ != mode) {
+    hyper_dmabuf_mode_ = mode;
+    if (!hyper_dmabuf_mode_) {
```

Comment:
why not if (hyper_dmabuf_mode_) so that else is not needed?

Reply:
Reasonable, updated the PR accordingly :-)